docs/getting-started/dockstore-workflows.rst

```diff
@@ -65,165 +124,67 @@ seamless.
-types of quick registration and one type of manual registration. These quick
-methods create workflow stubs, which are simply workflows that have not yet
-been confirmed to be real workflows. You can individually refresh a workflow
-stub that you identify as a real workflow in order to start pulling in workflow
-files and other data.
-
-To register content on Dockstore, you must have an account on Dockstore and
-link the necessary third-party accounts. Once this is done you can register
-workflows from the My Workflows page.
-
-Quick Registration via the Web UI
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-Quick registration is best used for workflows that follow the simple format
-that Dockstore suggests. It can still be used if your workflows are
-non-standard format, however there can be some drawbacks.
-
-Some users have multiple workflows within one Git repository, however each
-workflow entry on Dockstore only contains a single workflow. This is
-a problem as the Git path is used to uniquely identify a Dockstore workflow.
-The solution to this is to allow users to specify a workflow name that is
-appended to the Dockstore path. This would allow them to have multiple
-Dockstore workflows with the same Git repository. Quick registration does
-not allow you to create workflows with workflow names.
-To do that you must do manual registration, which is described later.
-
-Refresh All
-^^^^^^^^^^^
-Refresh all will look at all of your third-party accounts and do the following:
-
-- Create stub workflows for all Git repositories which do not exist on
-  Dockstore
-- Refresh all workflows that have been converted from stub to full workflows
-- Add user to any workflows that exist on Dockstore that they should have
-  access to
-
-To refresh all, select the refresh all icon button on the left side of the
-My Workflows page.
-
-.. note:: For existing organizations, you can select `Refresh Organization`
-          to perform a refresh all on a specific organization.
-
-One drawback of refresh all is that it creates a stub for every single
-repository that you have access to, whether or not it contains a workflow.
-The following approach is a bit less automated but allows for more control.
+process, though you can still register workflows with non-standard format by
+using manual registration.
 
 Quick Register
-^^^^^^^^^^^^^^
+^^^^^^^^^^^^^^^
 Quick register provides a flow that lets you browse the repositories you
-have access to and quickly create standard stub workflows. The benefit of
-this approach is that you get some automation without having lots of
-stubs created.
-
-You can access quick register by clicking the plus button on the My
-Workflows page. The flow of this process is shown in the screenshots
-below.
-
-.. figure:: /assets/images/docs/quick-register-step-1.png
-   :alt: Quick Register step 1
-
-   Choose the quick register option in the Register workflow wizard
+have access to and quickly create standard stub workflows. You can access 
```

Comment:
It was already like this, but might be useful to define what a stub vs full workflow is.

Reply:
Actually I think I'm just going to remove the word stub, we are migrating to no stubs (a stub is simply going to be a workflow without any versions).